### PR TITLE
glossarium:0.4.1

### PR DESCRIPTION
**Pull request by slashformotion**
I am submitting
- [ ] a new package
- [x] an update for a package

### 0.4.1

- Resolved an issue causing Glossarium to crash when all entries had a defined, non-empty group.

### CODE REVIEW

File: packages/preview/glossarium/0.4.1/LICENCE
Comment (elegaanz) [inline packages/preview/glossarium/0.4.1/LICENCE:1]: There is a typo in the file name, it should be `LICENSE` with a S.

Comment (slashformotion) [reply packages/preview/glossarium/0.4.1/LICENCE:1]: I wrote it in french 😂

Comment (slashformotion) [reply packages/preview/glossarium/0.4.1/LICENCE:1]: I will correct it this evening

Comment (elegaanz) [reply packages/preview/glossarium/0.4.1/LICENCE:1]: > I wrote it in french 😂

Yeah that's what I thought, I often do the same mistake :smile:

**Comment by elegaanz** [comment]
Thank you :)

**Pull request merged**